validate-differential-expression/README.md

```diff
@@ -0,0 +1,57 @@
+# refine.bio Examples: Validate Differential Expression
+
+These notebooks compare the differential expression results from a refine.bio dataset to an non-refine.bio dataset that both have a similar experimental set up.
+
+You may obtain differential expression results from your own data or a collaborator’s data.
+It is good practice to validate your findings in another dataset that measures a similar tissue with a similar experimental design.
+You can use refine.bio datasets and the steps in this example to quickly check for agreement with your results.
+
+**Example Workflow R Notebooks**
```

Comment:
Instead of listing these notebooks in chronological order like you have here, we should center this on the notebook analyzing the overlap.

From https://github.com/AlexsLemonade/refinebio-examples/issues/28#issuecomment-482581185:

> I'm imagining the README for this module would read like... if you have a list of differentially expressed genes from your own dataset, you can follow validation notebook X (steps 3-5) to check your results with a refine.bio datasets. See notebook Y (steps 1-2) for how we got this list.

I would move the `Using your own data` section up here and I think it needs a rewrite to be specific to this module because the 3 requirements are important.

The performing differential expression on a non-refine.bio dataset (side note: the phrasing non-refine.bio dataset is confusing) bit is an aside. We're only tracking it here so folks (including future us!) know what we did.

Reply:
Just to clarify that I understand what @jaclyn-taroni means in the above comment. Do you want to present the notebooks in a way that the user can choose to skip notebooks depending on what state their own data is in?

Also, will users have to do differential expression on data they obtained from refine.bio before they can run gene_DE_validate.nb.html?

Reply:
Users should not _have to_ look at the `author_processed_DE` notebook unless they are curious.

> Also, will users have to do differential expression on data they obtained from refine.bio before they can run gene_DE_validate.nb.html?

Yes, the purpose of this module is to demonstrate how to validate one's own differential expression results using refine.bio data -> you need results to validate them. The output of `author_processed_DE` here is essentially a _stand-in_ for their own results.

Reply:
Okay, then phrasing it like above makes sense. We should also link to how they can obtain the list for refine.bio data (i.e the differential expression example)

Reply:
They create the list in the gene_DE.Rmd notebook.

Reply:
Where are we left with this?

Reply:
What's a better term for the non-refine.bio dataset? 'example author data' ?